client/state/audio/actions.js

```diff
@@ -0,0 +1,14 @@
+
+/**
+ * Internal dependencies
+ */
+import {
+	AUDIO_PLAY,
+} from 'state/action-types';
+
+export function audioPlay( reference ) {
+	return {
+		type: AUDIO_PLAY,
+		reference,
+	};
+}
```

Comment:
fun fact! we're allowed to simplify the syntax here since all we're doing is returning an object from the function

```js
export const audioPlay = reference => ( {
	type: AUDIO_PLAY,
	reference,
} );
```